lib/frederick_api/v2/background_job.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module FrederickAPI
+  module V2
+    # V2 Frederick API async background job record, does not use a DB, just
+    # exists to provide a place to parse background job responses coming from
+    # API so that we can handle them properly.
+    class BackgroundJob < Resource
+      attr_accessor :meta
+      belongs_to :contact
```

Comment:
I was under the impression that we do, so that it lives under locations/contacts/background_jobs. I wanted to use polymorphic relationships, but I didn't see how they would be supported by the base library, so I put that off till we need it.

Reply:
I think you can remove it